### ISSUE TITLE
Update documentation 

### ISSUE THREAD

**Issue by msldiarra** [orhan/react-native-bottom-navigation#1]
Hello,

It seems `ReactBottomNavigationPackage` has been renamed to `BottomNavigationPackage` but remains in your documentation.

Plus the example project is not using any.

Thanks.

**Comment by ghost** [comment]
And in the third step says 'In your MainActivity.java,...'

It should be 'In your MainApplication.java', I think.

Thanks for this library.

**Comment by orhan** [comment]
Yes guys, the documentation is a little bit outdated, as I didn't have the time to address this yet, sorry!

I'll add a proper documentation soon .. :)

**Comment by santiblanko** [comment]
Thanks for your support.
